### PR TITLE
Add c3

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Our discussion about non python packages was resolved. Could you please review and merge. It fails only for python35 on window due to some system issues.

**Comment by isuruf** [comment]
> Our discussion about non python packages was resolved.

What discussion?

File: recipes/c3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{% set name = "c3" %}
+{% set version = "0.4.18" %}
+{% set sha256 = "1521bf046745c347b6ebd3cfd1adb568355e702ffb85d272f3b6c2e82ec8b0ee" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/c3js/c3/archive/v0.4.18.tar.gz
```
Comment (dougalsutherland) [inline recipes/c3/meta.yaml]: Template this string.

File: recipes/c3/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: True
```
Comment (dougalsutherland) [inline recipes/c3/meta.yaml]: `noarch: generic`

File: recipes/c3/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  run:
+    - nodejs
+    
+test:
+  imports:
```
Comment (dougalsutherland) [inline recipes/c3/meta.yaml]: Test that a file is installed.

Comment (sannykr) [reply recipes/c3/meta.yaml]: Thanks for the review. To use this package, we simply import the file(just like jquery) and I don't think any kind of import will work. Do you have any suggestion for testing?

Comment (dougalsutherland) [reply recipes/c3/meta.yaml]: http://conda-forge.readthedocs.io/en/latest/testing.html#simple-existence-tests

Comment (sannykr) [reply recipes/c3/meta.yaml]: Thanks. I will look at it.

**Comment by sannykr** [comment]
@dougalsutherland : I have made all the recommended changes. Please let me know if you have any other suggestion.

**Comment by dougalsutherland** [comment]
Okay, `npm list` is a good solution.

Not sure what's up with [the Windows errors](https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.17887/job/l2w22aheki6lyu7v#L363) though....

**Comment by sannykr** [comment]
@conda-forge/staged-recipe: Could you please review and merge.

**Pull request merged**